src/storage/resourceManager.ts

```diff
@@ -309,69 +311,87 @@ export class ResourceManager {
     } else if (isCCloud(topic)) {
       return this.getCCloudKafkaCluster(topic.environmentId, topic.clusterId);
     }
-    // TODO(shoup): add isDirect() check here?
     return null;
   }
 
   // SCHEMA REGISTRY
 
-  /** Cache all of the CCloud schema registries at once. */
-  async setCCloudSchemaRegistries(clusters: CCloudSchemaRegistry[]): Promise<void> {
-    const clustersByEnv: CCloudSchemaRegistryByEnv = new Map();
-    clusters.forEach((cluster) => {
-      clustersByEnv.set(cluster.environmentId, cluster);
+  getSchemaRegistryKey(connectionId: ConnectionId): WorkspaceStorageKeys {
+    if (connectionId === CCLOUD_CONNECTION_ID) {
+      return WorkspaceStorageKeys.CCLOUD_SCHEMA_REGISTRIES;
+    }
+    // Only CCLoud support to start with, but can easily expand in future.
+    throw new Error(`Unsupported connectionId ${connectionId} for schema registry key`);
```

Comment:
Do we expect this to go away once all the various methods are connection-agnostic (i.e. having one mega `getKey(id: ConnectionId, entity: IResourceBase)` that would create the `{connectionType}{entityName}` key)?